src/Cafe/HW/Latte/Core/LatteShaderCache.cpp

```diff
@@ -251,7 +256,27 @@ void LatteShaderCache_load()
 
 		fsc_close(fscfile);
 	}
+	//get game background loading image for DRC
+	TGAFILE file2{};
+	d_shaderCacheLoaderState.textureId = nullptr;
+
+	std::string drcTexPath = fmt::format("{}/meta/bootDRCTex.tga", CafeSystem::GetMlcStoragePath(CafeSystem::GetForegroundTitleId()));
+	sint32 status2;
+	auto fscfile2 = fsc_open(drcTexPath.c_str(), FSC_ACCESS_FLAG::OPEN_FILE | FSC_ACCESS_FLAG::READ_PERMISSION, &status2);
```

Comment:
I suggest reusing `status` and `fscFile` instead of defining a new set of variables. Although this is subjective and not necessary.
At the very least, you should avoid numbered variable names (status, status2) and use more descriptive names.

Reply:
I did actually did try this but it would result in an error (cant remember if it was a runtime or a compiler error).